.circleci/rahti-ci.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+instance=${1}
+
+if [ "${instance}" == "dev" ] ; then
+    tag1=circle
+    tag2=$(git rev-parse HEAD)
+    token=${rahti_dev_pod_delete_token}
+elif [ "${instance}" == "dev" ] ; then
+    tag1=release
+    tag2=$(git describe --abbrev=0 --tags)
```

Comment:
Are you sure this is right? If there is no tag for HEAD it will give you the name of the previous reachable tag. This will override the previous image built with that tag, which might not be what we want. Would it be better to use `git describe --tags --exact-match` and fail if there is no tag?

Reply:
Your command seems safer. But if we expect to have cases where the new tag does not point to the head of a branch, we could do something like:

tag=$(git describe --abbrev=0 --tags)
commit=$(git rev-list -n 1 $tag)
git checkout $commit
lein uberwar

Then war step might be better to split into two -> war-tag and war-master. What do you think? Or would this complicate things with no added value?

Reply:
checking out something else than circle is building for us seems wrong. when you push a new tag `foobar`, even if it doesn't point at the head of a branch, circle will build it

Reply:
So circle has checked out the tag for us already? Then I see no case where the tag would not point to the head of the branch. Anyways I see no downside using "git describe --tags --exact-match" command, so I'll swap to it